addons/project/views/project_views.xml

```diff
@@ -948,8 +955,10 @@
                     <field name="user_id" avatar_field="image_128" filters="1"/>
                     <field name="date_deadline"/>
                     <field name="project_id"/>
-                    <field name="priority" widget="priority"/>
-                    <field name="stage_id"/>
+                    <field name="priority" widget="priority" groups="project.group_project_user"/>
+                    <field name="priority" widget="priority" readonly="1" groups="!project.group_project_user"/>
+                    <field name="stage_id" groups="project.group_project_user"/>
+                    <field name="stage_id" readonly="1" groups="!project.group_project_user"/>
```

Comment:
I am not sure it is really needed since the internal user without any access to the project app, cannot access to that view. I think he can just access to the form view, right? 🤔

Reply:
hello @xavierbol ,
i have done the changes,
Thank you for the suggestion slightly_smiling_face